src/proof_generation/kore/convert.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from typing import overload
+
+import pyk.kore.syntax as pyk_kore
+from pyk.kllvm import load
+
+load  # Only needed to quench the autoformatter.
```

Comment:
@tothtamas28 This probably isn't the idiomatic way to load the llvm binding?

Reply:
For an integration test, you can just

```python
from pyk.kllvm import load  # noqa: F401
```

or

```python
import pyk.kllvm.load  # noqa: F401
```

This is because the code in the `load` module is executed on import, so the binding gets generated in a temporary folder, is imported, and cleaned up on exit.

For production code, this is not good enough, as you do not want to do the compilation each time this module is imported in a fresh interpreter. So you need to compile the bindings in advance as part of your build pipeline.

Here's an example:

Compile:

https://github.com/runtimeverification/evm-semantics/blob/4beaee3b63a663dc95e3151e9227d572cc019cef/kevm-pyk/src/kevm_pyk/kompile.py#L193

Import:

https://github.com/runtimeverification/evm-semantics/blob/4beaee3b63a663dc95e3151e9227d572cc019cef/kevm-pyk/src/kevm_pyk/interpreter.py#L10